### PR TITLE
Add MysqlGTIDSet.Add() and Minus() methods

### PR DESCRIPTION
**Pull request by ostinru**
Add MysqlGTIDSet.Add() and Minus() methods, so it will help users to estimate GTID sets that are stored in binlog file.

### CODE REVIEW

File: mysql/mysql_gtid.go [hunk 1]
```diff
@@ -227,6 +227,58 @@ func (s *UUIDSet) AddInterval(in IntervalSlice) {
...
+		minuend = s.Intervals[i]
+		if minuend.Start < lastSub.Stop {
+			n = append(n, Interval{lastSub.Stop, minuend.Stop})
+		} else {
+			n = append(n, s.Intervals[i])
+		}
+	}
+
+	s.Intervals = n
+	s.Intervals = s.Intervals.Normalize()
```
Comment (atercattus) [inline mysql/mysql_gtid.go:279]: `s.Intervals = n.Intervals.Normalize()`?

**Review by atercattus — CHANGES_REQUESTED** [review]
lgtm. But fix the tests and lint, please.

**Review by atercattus — APPROVED** [review]

**Comment by ostinru** [comment]
@atercattus, could you please trigger github workflow checks for this PR?
I have no permissions for doing this.

**Comment by atercattus** [comment]
Sorry, I forgot about this 🙂

**Comment by ostinru** [comment]
Hi, @atercattus it seems that this PR is ready to be merged. Could you please merge it?

**Pull request merged**